### PR TITLE
Add and remove book

### PR DESCRIPTION
**Pull request by GutemaG**
## Add and remove book

- Integrate redux to the app
- Add removing and and adding book
- Add Reducer
- Use Reducer
- Use useSelector hook

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const ADDING_BOOK = 'books/ADDING_BOOK';
+const REMOVING_BOOK = 'books/REMOVING_BOOK';
```
Comment (gemmen29) [inline src/redux/books/books.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions like:
```javascript
<app name>/<reducer name>/<action type>
```
for example:
```javascript
const REMOVE_BOOK = 'react-bookstore/book/REMOVE_BOOK';
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const CHECKING_STATUS = 'categories/CHECKING_STATUS';
```
Comment (gemmen29) [inline src/redux/categories/categories.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions also here.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @GutemaG, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (GutemaG) [reply src/redux/books/books.js]: @gemmen29 thank you!

**Review by topeogunleye — APPROVED** [review]
### Hi @GutemaG, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**